pom.xml

```diff
@@ -24,7 +24,7 @@
 
         <!-- tests -->
         <maven-surefire-plugin.version>3.2.5</maven-surefire-plugin.version>
-        <jib-maven-plugin.version>3.4.2</jib-maven-plugin.version>
+        <jib-maven-plugin.version>3.4.4</jib-maven-plugin.version>
```

Comment:
Spring version needs a bump

Reply:
Addressed in new commits, 3.4.1 sees the following issue occur in itests:
`Caused by: org.springframework.beans.BeanInstantiationException: Failed to instantiate [java.util.List]: Specified class is an interface`
So have settled to stick with 3.4.0 for now